docker/scripts/dev_start.sh

```diff
@@ -118,7 +137,11 @@ do
         shift
         source ${APOLLO_ROOT_DIR}/docker/scripts/restart_map_volume.sh \
             "${map_name}" "${VOLUME_VERSION}"
-    ;;
+        ;;
+    --stop)
```

Comment:
s/--stop/stop/ would be cleaner and aligned with our other scripts.

Reply:
Okay, fixed.